### DISCUSSION TITLE
Information requests should be separate from a CO

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5565] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Extending the Information Request to items other than loans is a good idea, but it is still tied to a collection object. We would like to be able to tally all of our requests in one place - some of our requests are inquiries about the collection (do you have specimens of X) that, if we don't have the requested materials, can't be tied to a collection object.

**Reported By**
UMich

**Comment by g1rly-c0d3r** [comment]
Linked in https://github.com/specify/specify7/issues/8179
